wgsl/index.bs

```diff
@@ -6411,7 +6426,137 @@ Issue: (dneto) Default rounding mode is an implementation choice.  Is that what
 Issue: Check behaviour of the f32 to f16 conversion for numbers just beyond the max normal f16 values.
 I've written what an NVIDIA GPU does.  See https://github.com/google/amber/pull/918 for an executable test case.
 
-# Memory Model TODO # {#memory-model}
+# Memory Model # {#memory-model}
+
+In general, [SHORTNAME] follows the [[!VulkanMemoryModel|Vulkan Memory Model]].
+The remainder of this section describes how [SHORTNAME] programs map to the
+Vulkan Memory Model.
```

Comment:
Maybe add a link to https://github.com/KhronosGroup/Vulkan-MemoryModel/blob/master/alloy/spirv.als for the formal model, as well as to the spec (which is already linked to in later sections)?

Reply:
The `!VulkanMemoryModel` is the link to the spec. I could add a link to the alloy model in a note perhaps.